test/templates/side-nav/index.njk

```diff
@@ -1,386 +1,783 @@
+{% set example %}
+<nav class="fd-side-nav">
+    <div class="fd-side-nav__main-navigation">
+      <ul class="test-sidenav-list">
+        <li>Item 1</li>
+        <li>Item 1</li>
+        <li>Item 1</li>
+        <li>Item 1</li>
+      </ul>
+    </div>
+    <div class="fd-side-nav__utility">
+       <ul class="test-sidenav-list">
+        <li>Item 1</li>
+        <li>Item 1</li>
+       </ul>
+    </div>
+</nav>
+{% endset %}
+{{ format(example) }}
 
-        {{ side_nav(
-                properties = {
-                    "items": [
-                    { "label": "Link Item"},
-                    { "label": "Link Item" },
-                    {
-                        "label": "item link",
-                        "selected": true
-                     },
-                    { "label": "Link Item" },
-                    { "label": "Link Item" },
-                    { "label": "Link Item" },
-                    { "label": "Link Item" }
-                    ]
-                }
-            )
-        }}
-    {% endset %}
-    {{ format(example) }}
+<br><br><br>
+<h2>Side navigation with icons - condensed state</h2>
+{% set example %}
+<nav class="fd-side-nav fd-side-nav--condensed">
+   <div class="fd-side-nav__main-navigation">
+      <ul class="test-sidenav-list">
+        <li class="test-sidenav-list-icon sap-icon--home"></li>
+        <li class="test-sidenav-list-icon sap-icon--calendar"></li>
+        <li class="test-sidenav-list-icon sap-icon--employee"></li>
+        <li class="test-sidenav-list-icon sap-icon--activities"></li>
+      </ul>
+    </div>
+    <div class="fd-side-nav__utility">
+       <ul class="test-sidenav-list">
+        <li class="test-sidenav-list-icon sap-icon--bar-chart"></li>
+        <li class="test-sidenav-list-icon sap-icon--video"></li>
+       </ul>
+    </div>
+</nav>
+{% endset %}
+{{ format(example) }}
 
-    <br><br><br>
-<h2>w/ group headings</h2>
-    {% set example %}
-    {{ side_nav(
-        properties = {
-          "title": "Group Title",
-          "navgroups": [
-            {
-              "title": "Group Name",
-              "items": [
-                { "label": "icon link"},
-                {
-                    "label": "item link",
-                    "haschild": true
-                 },
-                { "label": "icon link"},
-                { "label": "item link" },
-                { "label": "item link" }
-              ]
-            },
-            {
-                "title": "Group Name",
-                "items": [
-                  { "label": "item link" },
-                  {
-                      "label": "item link",
-                      "haschild": true
-                   },
-                  { "label": "item link" },
-                  { "label": "item link" },
-                  { "label": "item link" }
-              ]
-            }
-          ]
-        }
-    )}}
-    {% endset %}
-    {{ format(example) }}
+<h2>Nested List Without Icons</h2>
```

Comment:
All these nested list examples should be in their own page:  `test/templates/nested-list/index.njk`.  This file should only have the examples that show the `fd-side-nav*` classes.